plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -1,4 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.4"?>
 <plugin>
+  <extension-point id="com.google.cloud.tools.eclipse.appengine.libraries" name="App Engine Libraries" schema="schema/com.google.cloud.tools.eclipse.appengine.libraries.exsd"/>
+  
+  <extension point="com.google.cloud.tools.eclipse.appengine.libraries">
+    <library 
+          id="com.google.cloud.tools.eclipse.appengine.libraries/appengine-api"
+          name="App Engine API">
+      <jarFile>
+        <mavenCoordinates
+              artifactId="appengine-api-1.0-sdk"
+              groupId="com.google.appengine"
+              repositoryUri="http://repo1.maven.org/maven2/">
```

Comment:
My concern here is that we want to define the repo to be used, not relying on what is defined in the users' settings.xml where they can override what is central, or define a mirror for it. But to allow for user customization, probably we should allow them to override it in the CT4E preferences.